web/src/narrow.js

```diff
@@ -194,12 +196,13 @@ export function activate(raw_operators, opts) {
     }
 
     const coming_from_recent_topics = recent_topics_util.is_visible();
+    const coming_from_inbox = inbox_util.is_visible();
 
     // The empty narrow is the home view; so deactivate any narrow if
     // no operators were specified. Take us to all messages when this
     // happens from recent conversations view.
     if (raw_operators.length === 0) {
-        deactivate(coming_from_recent_topics);
+        deactivate(coming_from_recent_topics || coming_from_inbox);
```

Comment:
The parameters here don't seem correctly updated for the inversion of meaning in the `deactivate` code below. Probably this would be a good case for a prep commit to invert the `deactivate` parameter semantics.

Reply:
Done.